CONTRIBUTING.md

```diff
@@ -69,16 +69,23 @@ We provide a script that can automatically create all the necessary files for yo
   - Use existing Sass variables for spacing (margin, padding, sizing, etc).
   - Use [`themed`](https://github.com/iTwin/iTwinUI/blob/main/src/style/theme.scss#L430) mixin wherever color is involved.
 - Define classes for your mixin in `src/[component-name]/classes.scss` file.
-  - *Running the `createComponent` command will do this for you.*
+  - _Running the `createComponent` command will do this for you._
```

Comment:
What it does? Cause I can't see the difference 😀

Reply:
Right, there is no visual difference. This change is also not by me! It was by the linter/formatter that runs after saving/committing the file.